Cabal/Distribution/PackageDescription/Parse.hs

```diff
@@ -161,6 +162,14 @@ storeXFieldsPD _ _ = Nothing
 -- ---------------------------------------------------------------------------
 -- The Library type
 
+subLibFieldDescrs :: [FieldDescr Library]
+subLibFieldDescrs =
+  [ simpleField "library"
+                        showToken             parseTokenQ
+                        (fromMaybe "subLibFieldDescrs" . libName)
```

Comment:
I'm... not sure if we should `cabalBug` this. I cargo-culted the stanza header from `executableFieldDescrs`, and I don't know if this isn't syntax that people are actually using.